apps/zotonic_mod_auth2fa/src/support/z_auth2fa_qrcode_demo.erl

```diff
@@ -110,7 +110,7 @@ totp(Key, Period) ->
 
 %% @doc RFC-4226 "HOTP: An HMAC-Based One-Time Password Algorithm" http://tools.ietf.org/html/rfc4226
 hotp(Key, Count) when is_binary(Key), is_integer(Count) ->
-	HS = z_utils:hmac(sha, Key, <<Count:64>>),
+	HS = crypto:mac(hmac, sha, Key, <<Count:64>>),
```

Comment:
Should these ones stay like this or also moved to `z_crypto`? I think I can also understand why it should stay here.

Reply:
This is the fixed algorithm, so better keep it here.